### PR TITLE
Login / Sign up pages

### PR DESCRIPTION
**Pull request by HarshnaNHaswani**
Kindly, review

### CODE REVIEW

File: cart.html [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+
+
+
+      <li class="list-item">
+        <a href="/index-user.html">Home</a>
+      </li>
+
+
+
+
```
Comment (hiteshverma27) [inline cart.html:54]: Use a code formatter once before pushing

**Review by hiteshverma27 — APPROVED** [review]
Rest looks good

File: cart.html (hunk 1 above)
Comment (HarshnaNHaswani) [reply cart.html:54]: I do, but it doesn't remve extra lines

**Comment by HarshnaNHaswani** [comment]
Thank you @hiteshverma27  for your time and the review, will format the files properly

**Comment by ashifkhn** [comment]
Looks good. Good to go.

**Review by ashifkhn — APPROVED** [review]
Good to merge.

**Pull request merged**